modules/sno-du-configuring-sriov.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-sriov_{context}"]
+= Configuring single root I/O virtualization (SR-IOV)
+
+SR-IOV is commonly used to enable the fronthaul and the midhaul networks.
+
+Procedure
+
+* Use the following configuration to configure SRIOV on a single node distributed unit (DU). Note that the first custom resource (CR) is required. The following CRs are examples.
```

Comment:
Need numbered step. 

Create a `SriovOperatorConfig` CR similar to the following:
Edit the `SriovOperatorConfig` CR similar to the following:

Reply:
The user uses the example to create the file.